bridge/cypress/integration/evaluations.spec.ts

```diff
@@ -42,6 +46,39 @@ describe('evaluations', () => {
 
     cy.visit('/project/sockshop/service/carts/context/da740469-9920-4e0c-b304-0fd4b18d17c2/stage/staging');
 
-    cy.byTestId('keptn-evaluation-details-scoreInfo').should('have.text', '33.99');
+    cy.byTestId('keptn-evaluation-details-scoreInfo').should('have.text', '33.99 < 75');
+  });
+
+  it('should show key sli info', () => {
+    cy.intercept('GET', '/api/project/sockshop/serviceStates', {
+      statusCode: 200,
+      fixture: 'get.sockshop.service.states.mock.json',
+    });
+    cy.intercept('GET', '/api/project/sockshop/deployment/da740469-9920-4e0c-b304-0fd4b18d17c2', {
+      statusCode: 200,
+      fixture: 'get.sockshop.service.carts.deployment.mock.json',
+    });
+    cy.intercept('GET', 'api/mongodb-datastore/event/type/sh.keptn.event.evaluation.finished?*', {
+      statusCode: 200,
+      fixture: 'get.sockshop.service.carts.evaluations.keysli.mock.json',
+    });
+
+    cy.visit('/project/sockshop/service/carts/context/da740469-9920-4e0c-b304-0fd4b18d17c2/stage/staging');
+    cy.byTestId('keptn-service-view-service-carts').should('exist');
+    cy.get('ktb-heatmap').should('exist');
+
+    cy.byTestId('keptn-evaluation-details-scoreInfo').should('have.text', '50 < 75');
+    cy.byTestId('keptn-evaluation-details-resultInfo').should('have.text', 'Result: fail');
```

Comment:
Could you move most of the tests inside page objects?
```ts
public assertScoreInfo(score: number, equality: '<' | '<=' | '>' | '>=', threshold: number){...}
public assertResultInfo(type: ResultType){...}
public assertKeySliInfo(type: 'passed' | 'failed'){...}
...
```

Reply:
I also reused some of the page object intercepts :)